.azure-pipelines/platform_ci_dev_pipeline.yml

```diff
@@ -5,7 +5,7 @@ parameters:
  - name: use_case_base_path
    displayName: "type of model to execute"
  - name: deployment_type
-   displayName: "determine type of deployment - aks, aml, docker, webapp"
```

Comment:
`"docker"` seems not to be a supported option anymore, I can remove it as part of this PR if that's the case